index.html

```diff
@@ -11175,13 +11175,17 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-controls">
 			<pdef>aria-controls</pdef>
 			<div class="property-description">
-				<p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the current element. See related <pref>aria-owns</pref>.</p>
-				<p>For example:</p>
+				<p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the current element or composite widget. See related <pref>aria-details</pref>.</p>
```

Comment:
Am I right in reading this as saying that `aria-controls` should sometimes be added to the composite widget container role, and not to the specific interactable/focusable child element? I know it's technically allowed, my two thoughts on it are just:
1. Does it do anything in practice? (Or if it doesn't now, does that mapping exist as a pattern on native platforms & is that the pattern we want to try to convince AT to support in the future?)
2. Would it make sense to be explicit here about which element `aria-controls` should go on? It's allowed on all elements of base markup so that's not super helpful, and the new wording is a bit more ambiguous than the current wording, at least to me.

Reply:
the composite widget bit is kinda important for things like listboxes, radiogroups, menus - where focus could be on the composite widget and never reach a child element.  maybe if that's just clarified its meant to be for those use cases?